### DISCUSSION TITLE
[Not an Issue/Tech question] Is bukkit plugins allowed to access mod entities?

### DISCUSSION THREAD

**Discussion by FabioZumbi12** [IzzelAliz/Arclight#1495] [General]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-forge-1.20.1-1.0.5

### OS & Java versions

jdk-17

### Plugins and Mods

```raw
Plugins (9): LuckPerms, Vault, Multiverse-Core, Multiverse-NetherPortals, Essentials, EssentialsSpawn, EssentialsGeoIP, RedProtect, PlugManX (PlugMan)

Mod List:
• minecraft server-1.20.1-20230612.114412-srg.jar : minecraft (1.20.1) - 1
• minecraft forge-1.20.1-47.2.20-universal.jar : forge (47.2.20) - 1
• mods_folder Vampirism-1.20.1-1.10.8.jar : vampirism (1.10.8) - 2
• arclight arclight-1.20.1-1.0.5-1a8925b.jar : arclight (1.20.1-1.0.5-1a8925b) - 1
```

### Description

Is bukkit plugins (like antigrief plugins) allowed to access mod entities, like using from `EntityType.values()`?
If yes, theres an example?

**Comment by qyl27** [comment]
Yes, you may use `valueOf` to get the enum value, although it wasn't defined in Bukkit API.

**Comment by qyl27** [comment]
Btw, we recommend having questions in developing or using our discussions page, but not issues.  
https://github.com/IzzelAliz/Arclight/discussions

**Comment by FabioZumbi12** [comment]
I got a help from discord where using `Registry.ENTITY_TYPE.stream()` it worked for me, exposing all loaded entities, and it seems this method its from Bukkit.
